test/scanner/Tests/ScanLetSpec.hs

```diff
@@ -95,32 +137,121 @@ spec = do
     scanTokenTypes input `shouldBe` Right output
 
   it "supports explicit braces around definition" $ do
-    pendingWith "help needed"
-    -- let a = { a_block_1_1
-    --  a_block_1_2 ;
-    --      a_block_2_1
-    --        a_block_2_2 }
+    let input = unlines
+          [ "let a = { a_block_1_1"
+          , "            a_block_1_2;"
+          , "          a_block_2_1"
+          , "            a_block_2_2 }"
+          , "outside_let"
+          ]
+        output =
+          [ TLet
+          , TLbrace
+          , TId "a"
+          , TEq
+          , TLbrace
+          , TLbrace -- Inserts extra brace pair?
```

Comment:
The extra brace pair is due to the fact that, at the moment, `=` starts a block + there's a bug in the scanner where it doesn't check if the user already explicitly started a block before starting one itself.

Technically not an issue on its own because `{x}` is just parsed as `x` now, but there are further problems where the scanner doesn't actually end the blocks it starts at the right place.